lib/realtime_web/controllers/fallback_controller.ex

```diff
@@ -54,4 +54,13 @@ defmodule RealtimeWeb.FallbackController do
     |> put_view(RealtimeWeb.ChangesetView)
     |> render("error.json", changeset: changeset)
   end
+
+  def call(conn, response) do
+    Helpers.log_error("UnknownError",response)
+
+    conn
+    |> put_status(:not_found)
```

Comment:
this might be better as a 422 and not a 404

Reply:
That is for the TenantController for when tenants are not found though

Reply:
Oh...